pr_agent/algo/pr_processing.py

```diff
@@ -19,26 +20,33 @@
 PATCH_EXTRA_LINES = 3
 
 
-def get_pr_diff(git_provider: Union[GithubProvider, Any], token_handler: TokenHandler) -> str:
+def get_pr_diff(git_provider: Union[GithubProvider, Any], token_handler: TokenHandler,
```

Comment:
Consider refactoring the `get_pr_diff` function to avoid the use of global variables. Instead of modifying the global `PATCH_EXTRA_LINES` variable, you could pass it as a parameter to the functions that need it. This would make the code more modular and easier to test. [important]